common/lib/acm-provider.php

```diff
@@ -67,10 +67,10 @@ public function action_do_robotstxt() {
 		if ( ! is_array( $disallowed ) || empty( $disallowed ) )
 			return;
 
-		echo 'User-agent: ' . $this->crawler_user_agent . PHP_EOL;
+		echo esc_textarea( 'User-agent: ' . $this->crawler_user_agent . PHP_EOL );
```

Comment:
Why esc_textarea()?

Reply:
I was thinking becase this is `echo`ed to robots.txt.
Should I stick to `esc_html`?